ui/client/reducers/httpErrorHandler.js

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
do we really want it as seperate reducer?

Reply:
In future we can do handler for all type of errors.